### PR TITLE
[MRG, ENH] Change the sEEG tutorial to support new data

### PR DESCRIPTION
**Pull request by alexrockhill**
Address new data in https://github.com/mne-tools/mne-misc-data/pull/5.

Ok, there is a bit of an issue with the coordinate transforms that I am still working out, but I think everything otherwise is okay.

### CODE REVIEW

**Comment by alexrockhill** [comment]
Ok this should be good to go once the CIs pass!

**Comment by alexrockhill** [comment]
The artifact looks good
https://29553-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/clinical/20_seeg.html

**Comment by drammock** [comment]
this part is not formatted right:
Screenshot_2021-06-15 Working with sEEG data — MNE 0 24 dev0 documentation

**Comment by alexrockhill** [comment]
> this part is not formatted right:
> Screenshot_2021-06-15 Working with sEEG data — MNE 0 24 dev0 documentation

Good catch! I was too focused on the plots turning out alright.

**Comment by alexrockhill** [comment]
Ok fixed the note and my other note that I modeled it off that was also a mistake... oops

https://29569-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/clinical/20_seeg.html?highlight=seeg

File: tutorials/clinical/20_seeg.py [hunk 1]
```diff
@@ -42,6 +43,8 @@
 
 print(__doc__)
 
+np.set_printoptions(suppress=True)  # suppress scientific notation
```
Comment (agramfort) [inline tutorials/clinical/20_seeg.py:46]: is this really necessary?

**Review by agramfort — APPROVED** [review]
besides LGTM

thx @alexrockhill

File: tutorials/clinical/20_seeg.py (hunk 1 above)
Comment (larsoner) [reply tutorials/clinical/20_seeg.py:46]: I'm okay with having this, but we need to update `reset_warnings` to `np.set_printoptions` to whatever they are when `conf.py` is initially loaded (which can be obtained before any examples are run directly in `conf.py` using `get_printoptions` presumably)

Comment (alexrockhill) [reply tutorials/clinical/20_seeg.py:46]: I have a hard time understanding the affines in scientific notation. I'm not sure what you mean @larsoner, is that just like putting it back at the end?

Also that's how `nibabel` does it and it looks nice.

**Comment by larsoner** [comment]
I'll push a quick commit

**Comment by alexrockhill** [comment]
Well I get what you did I think but I'm definitely glad you did it :) Thanks

**Comment by larsoner** [comment]
Go go go @alexrockhill !

**Pull request merged**